index.html

```diff
@@ -102,75 +115,96 @@ <h2 class="mx-1rem">Question:</h2>
           </ul>
         </div>
       </section>
-      <section class="js-bookmarks hidden">
-        <div class="card">
+      <section class="hidden" data-js="bookmarks">
+        <div class="card" data-js="card">
           <button class="card__bookmark">
             <i
-              class="
-                card__bookmark card__bookmark-icon
-                fas
-                fa-bookmark
-                js-bookmark
-              "
+              class="card__bookmark card__bookmark-icon fas fa-bookmark"
+              data-js="bookmark"
             ></i>
           </button>
           <h2 class="mx-1rem">Question:</h2>
-          <p class="card__question mx-1rem">
+          <p class="card__question-text mx-1rem">
             Lorem, ipsum dolor sit amet consectetur adipisicing elit. Facilis
             quaerat asperiores, eius alias doloribus ut doloremque laudantium
             soluta ipsum enim!
           </p>
-          <button class="card__toggle-answer-button">Show Answer</button>
+          <button class="card__toggle-answer-button" data-js="toggle-answer">
```

Comment:
Don't know if a would prefere a notion like "answer__toggle-button", "answer__heading" and "answer__text" to fit better in the BEM schema. But on the other hand its part of the card, so i'm not sure at all.

Reply:
It is a good point, thank you! I think both ways would their arguments and are readable.